benchmark/output_formatter.cpp

```diff
@@ -5,7 +5,8 @@
 // (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
 
 #include "output_formatter.hpp"
-#include "random.hpp"
+
+#include <cmath>
```

Comment:
is this required?

Reply:
clang-18 gets very annoyed if I don't add all headers in use by the unit... but yes.